docs/synthetics/private-locations.md

```diff
@@ -0,0 +1,37 @@
+title: Private Locations
+description: Private locations allows you to monitor the APIs and webpages that are not accessible from the Internet such as your internal websites, private APIs, etc
+
+Private locations allow you to run the synthetics tests from your private network. Some of the use cases for using private locations are:
+
+* Monitor internal APIs or web applications that are not accessible from the Internet
+* Compare the performance of your applications between the Internet & your private network
+* Monitor your applications from specific locations that are not supported by Sematext
+
+## Create Private Location
+
+To create a private location, navigate to Synthetics -> Private Locations -> New Private Location. 
+
+![Create Private Location]()
+
+* **Name** - Unique name for the private location
+* **Description** - A brief description of the location
+* **City** - Select the city where the machine running the agent is physically located. This is used to calculate latitude/longitude to plot the location in the map. If your city is not listed, select the nearest city.
```

Comment:
This is used to calculate latitude/longitude to plot the location in the map. If your city is not listed, select the nearest city. => If your city is not listed, select the nearest city. This is used to calculate latitude/longitude to plot the location in the map. 
(just swapping sentences, a little easier for reader to follow)